team-validator.js

```diff
@@ -214,6 +215,14 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		if (tools.getNature(set.nature).id && !tools.getNature(set.nature).name.exists) {
```

Comment:
We need the name and id of ``set.nature``. It can be simplified to ``if ((set.nature).id && !(set.nature).name.exists)`` now that you mention it though.